djangoplicity/archives/base.py

```diff
@@ -800,3 +801,16 @@ def get_object_identifier_for_pk( cls, pk='' ):
         """
         return str( pk )
         #return "%s.%s:%s" % ( self._meta.app_label, self._meta.model_name, str( self.pk ) )
+
+
+class UnicodeMixin(object):
+
+  """Mixin class to handle defining the proper __str__/__unicode__
+  methods in Python 2 or 3."""
+
+  if sys.version_info[0] >= 3: # Python 3
+      def __str__(self):
+          return self.__unicode__()
+  else:  # Python 2
+      def __str__(self):
+          return self.__unicode__().encode('utf8')
```

Comment:
I found this in the below links,  what is your opinion @EdisonArango?

https://lucumr.pocoo.org/2011/1/22/forwards-compatible-python/, https://docs.python.org/3.3/howto/pyporting.html